2019-08-28 22:24:30,873-common.basepage-INFO-日志信息:等待元素可见- [basepage.py:53 ]
2019-08-28 22:24:30,874-common.basepage-ERROR-日志信息:等待元素可见失败- [basepage.py:62 ]
Traceback (most recent call last):
  File "/Users/liliang/PycharmProjects/web_autotest1/common/basepage.py", line 56, in visible_element
    WebDriverWait(self.driver,timeout,frequency).until(EC.visibility_of_element_located(loc))
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/support/wait.py", line 68, in until
    end_time = time.time() + self._timeout
TypeError: unsupported operand type(s) for +: 'float' and 'str'
2019-08-28 22:24:31,336-common.basepage-INFO-日志信息:失败截图成功,图片存放在/Users/liliang/PycharmProjects/web_autotest1/Outputs/screenshoots目录下- [basepage.py:37 ]
2019-08-28 22:24:31,615-common.basepage-INFO-日志信息:等待元素可见- [basepage.py:53 ]
2019-08-28 22:24:31,615-common.basepage-ERROR-日志信息:等待元素可见失败- [basepage.py:62 ]
Traceback (most recent call last):
  File "/Users/liliang/PycharmProjects/web_autotest1/common/basepage.py", line 56, in visible_element
    WebDriverWait(self.driver,timeout,frequency).until(EC.visibility_of_element_located(loc))
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/support/wait.py", line 68, in until
    end_time = time.time() + self._timeout
TypeError: unsupported operand type(s) for +: 'float' and 'str'
2019-08-28 22:24:32,016-common.basepage-INFO-日志信息:失败截图成功,图片存放在/Users/liliang/PycharmProjects/web_autotest1/Outputs/screenshoots目录下- [basepage.py:37 ]
2019-08-28 22:24:32,295-common.basepage-INFO-日志信息:等待元素可见- [basepage.py:53 ]
2019-08-28 22:24:32,296-common.basepage-ERROR-日志信息:等待元素可见失败- [basepage.py:62 ]
Traceback (most recent call last):
  File "/Users/liliang/PycharmProjects/web_autotest1/common/basepage.py", line 56, in visible_element
    WebDriverWait(self.driver,timeout,frequency).until(EC.visibility_of_element_located(loc))
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/support/wait.py", line 68, in until
    end_time = time.time() + self._timeout
TypeError: unsupported operand type(s) for +: 'float' and 'str'
2019-08-28 22:24:32,694-common.basepage-INFO-日志信息:失败截图成功,图片存放在/Users/liliang/PycharmProjects/web_autotest1/Outputs/screenshoots目录下- [basepage.py:37 ]
2019-08-28 22:24:32,906-common.basepage-INFO-日志信息:等待元素可见- [basepage.py:53 ]
2019-08-28 22:24:32,907-common.basepage-ERROR-日志信息:等待元素可见失败- [basepage.py:62 ]
Traceback (most recent call last):
  File "/Users/liliang/PycharmProjects/web_autotest1/common/basepage.py", line 56, in visible_element
    WebDriverWait(self.driver,timeout,frequency).until(EC.visibility_of_element_located(loc))
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/support/wait.py", line 68, in until
    end_time = time.time() + self._timeout
TypeError: unsupported operand type(s) for +: 'float' and 'str'
2019-08-28 22:24:33,312-common.basepage-INFO-日志信息:失败截图成功,图片存放在/Users/liliang/PycharmProjects/web_autotest1/Outputs/screenshoots目录下- [basepage.py:37 ]
2019-08-28 22:24:33,595-common.basepage-INFO-日志信息:等待元素可见- [basepage.py:53 ]
2019-08-28 22:24:33,596-common.basepage-ERROR-日志信息:等待元素可见失败- [basepage.py:62 ]
Traceback (most recent call last):
  File "/Users/liliang/PycharmProjects/web_autotest1/common/basepage.py", line 56, in visible_element
    WebDriverWait(self.driver,timeout,frequency).until(EC.visibility_of_element_located(loc))
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/support/wait.py", line 68, in until
    end_time = time.time() + self._timeout
TypeError: unsupported operand type(s) for +: 'float' and 'str'
2019-08-28 22:24:34,119-common.basepage-INFO-日志信息:失败截图成功,图片存放在/Users/liliang/PycharmProjects/web_autotest1/Outputs/screenshoots目录下- [basepage.py:37 ]
2019-08-28 22:24:34,177-common.basepage-INFO-日志信息:等待元素可见- [basepage.py:53 ]
2019-08-28 22:24:34,177-common.basepage-ERROR-日志信息:等待元素可见失败- [basepage.py:62 ]
Traceback (most recent call last):
  File "/Users/liliang/PycharmProjects/web_autotest1/common/basepage.py", line 56, in visible_element
    WebDriverWait(self.driver,timeout,frequency).until(EC.visibility_of_element_located(loc))
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/support/wait.py", line 68, in until
    end_time = time.time() + self._timeout
TypeError: unsupported operand type(s) for +: 'float' and 'str'
2019-08-28 22:24:34,179-common.basepage-ERROR-日志信息:截取网页截图失败- [basepage.py:39 ]
Traceback (most recent call last):
  File "/Users/liliang/PycharmProjects/web_autotest1/common/basepage.py", line 56, in visible_element
    WebDriverWait(self.driver,timeout,frequency).until(EC.visibility_of_element_located(loc))
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/support/wait.py", line 68, in until
    end_time = time.time() + self._timeout
TypeError: unsupported operand type(s) for +: 'float' and 'str'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/liliang/PycharmProjects/web_autotest1/common/basepage.py", line 36, in save_web_screenshot
    self.driver.save_screenshot(screenshoots_dir + '/' + filepath)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 1055, in save_screenshot
    return self.get_screenshot_as_file(filename)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 1032, in get_screenshot_as_file
    png = self.get_screenshot_as_png()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 1064, in get_screenshot_as_png
    return base64.b64decode(self.get_screenshot_as_base64().encode('ascii'))
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 1074, in get_screenshot_as_base64
    return self.execute(Command.SCREENSHOT)['value']
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=76.0.3809.100)

2019-08-28 22:24:34,184-common.basepage-INFO-日志信息:等待元素可见- [basepage.py:53 ]
2019-08-28 22:24:34,184-common.basepage-ERROR-日志信息:等待元素可见失败- [basepage.py:62 ]
Traceback (most recent call last):
  File "/Users/liliang/PycharmProjects/web_autotest1/common/basepage.py", line 56, in visible_element
    WebDriverWait(self.driver,timeout,frequency).until(EC.visibility_of_element_located(loc))
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/support/wait.py", line 68, in until
    end_time = time.time() + self._timeout
TypeError: unsupported operand type(s) for +: 'float' and 'str'
2019-08-28 22:24:34,187-common.basepage-ERROR-日志信息:截取网页截图失败- [basepage.py:39 ]
Traceback (most recent call last):
  File "/Users/liliang/PycharmProjects/web_autotest1/common/basepage.py", line 56, in visible_element
    WebDriverWait(self.driver,timeout,frequency).until(EC.visibility_of_element_located(loc))
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/support/wait.py", line 68, in until
    end_time = time.time() + self._timeout
TypeError: unsupported operand type(s) for +: 'float' and 'str'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/liliang/PycharmProjects/web_autotest1/common/basepage.py", line 36, in save_web_screenshot
    self.driver.save_screenshot(screenshoots_dir + '/' + filepath)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 1055, in save_screenshot
    return self.get_screenshot_as_file(filename)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 1032, in get_screenshot_as_file
    png = self.get_screenshot_as_png()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 1064, in get_screenshot_as_png
    return base64.b64decode(self.get_screenshot_as_base64().encode('ascii'))
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 1074, in get_screenshot_as_base64
    return self.execute(Command.SCREENSHOT)['value']
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=76.0.3809.100)

2019-08-28 22:24:34,190-common.basepage-INFO-日志信息:等待元素可见- [basepage.py:53 ]
2019-08-28 22:24:34,191-common.basepage-ERROR-日志信息:等待元素可见失败- [basepage.py:62 ]
Traceback (most recent call last):
  File "/Users/liliang/PycharmProjects/web_autotest1/common/basepage.py", line 56, in visible_element
    WebDriverWait(self.driver,timeout,frequency).until(EC.visibility_of_element_located(loc))
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/support/wait.py", line 68, in until
    end_time = time.time() + self._timeout
TypeError: unsupported operand type(s) for +: 'float' and 'str'
2019-08-28 22:24:34,192-common.basepage-ERROR-日志信息:截取网页截图失败- [basepage.py:39 ]
Traceback (most recent call last):
  File "/Users/liliang/PycharmProjects/web_autotest1/common/basepage.py", line 56, in visible_element
    WebDriverWait(self.driver,timeout,frequency).until(EC.visibility_of_element_located(loc))
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/support/wait.py", line 68, in until
    end_time = time.time() + self._timeout
TypeError: unsupported operand type(s) for +: 'float' and 'str'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/liliang/PycharmProjects/web_autotest1/common/basepage.py", line 36, in save_web_screenshot
    self.driver.save_screenshot(screenshoots_dir + '/' + filepath)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 1055, in save_screenshot
    return self.get_screenshot_as_file(filename)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 1032, in get_screenshot_as_file
    png = self.get_screenshot_as_png()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 1064, in get_screenshot_as_png
    return base64.b64decode(self.get_screenshot_as_base64().encode('ascii'))
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 1074, in get_screenshot_as_base64
    return self.execute(Command.SCREENSHOT)['value']
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=76.0.3809.100)

